x-pack/plugins/aiops/public/components/change_point_detection/change_point_type_filter.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type FC, useCallback, useMemo } from 'react';
+import {
+  EuiComboBox,
+  type EuiComboBoxOptionOption,
+  type EuiComboBoxOptionsListProps,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiIcon,
+  EuiToolTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { isDefined } from '@kbn/ml-is-defined';
+import { type ChangePointType } from './change_point_detection_context';
+
+export type ChangePointUIValue = ChangePointType | undefined;
+
+interface ChangePointTypeFilterProps {
+  value: ChangePointType[] | undefined;
+  onChange: (changePointType: ChangePointType[] | undefined) => void;
+}
+
+const changePointTypes: Array<{ value: ChangePointType; description: string }> = [
+  {
+    value: 'dip',
+    description: i18n.translate('xpack.aiops.changePointDetection.dipDescription', {
+      defaultMessage: 'a significant dip occurs at this change point',
+    }),
+  },
+  {
+    value: 'spike',
+    description: i18n.translate('xpack.aiops.changePointDetection.spikeDescription', {
+      defaultMessage: 'a significant spike occurs at this point',
+    }),
+  },
+  {
+    value: 'distribution_change',
+    description: i18n.translate('xpack.aiops.changePointDetection.distributionChangeDescription', {
+      defaultMessage: 'the overall distribution of the values has changed significantly',
+    }),
+  },
+  {
+    value: 'step_change',
+    description: i18n.translate('xpack.aiops.changePointDetection.stepChangeDescription', {
+      defaultMessage:
+        'the change indicates a statistically significant step up or down in value distribution',
+    }),
+  },
+  {
+    value: 'trend_change',
+    description: i18n.translate('xpack.aiops.changePointDetection.trendChangeDescription', {
+      defaultMessage: 'there is an overall trend change occurring at this point',
```

Comment:
```suggestion
      defaultMessage: 'An overall trend change occurs at this point.',
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/150308/commits/26a638820227f69f2ba519ad6127e84a43dbcfcc